x-pack/plugin/esql/qa/testFixtures/src/main/resources/chunk.csv-spec

```diff
@@ -0,0 +1,96 @@
+###############################################
+# Tests for Chunk function
+#
+
+chunkExample
+required_capability: chunk_function
+
+// tag::chunk-with-field[]
+FROM books
+| EVAL chunks = CHUNK(description, 1, 20)
```

Comment:
what happens when the field is multivalued? e.g. description is `["foo", "bar", ...]`?
would be nice to get a test for it.